packages/expo-contacts/ios/ContactsModule.swift

```diff
@@ -128,11 +144,31 @@ public class ContactsModule: Module {
 
       controller.onViewDisappeared = {
         promise.resolve()
+        self.contactManipulationPromise = nil
       }
 
+      contactManipulationPromise = promise
       parent?.present(navController, animated: animated)
     }.runOnQueue(.main)
 
+    AsyncFunction("presentContactPickerAsync") { (promise: Promise) in
+      if currentContactPickingContext != nil {
+        throw ContactPickingInProgressException()
+      }
+
+      let pickerController = CNContactPickerViewController()
+
+      contactPickerDelegate = ContactPickerControllerDelegate(onContactPickingResultHandler: self)
+
+      currentContactPickingContext = ContactPickingContext(promise: promise)
```

Comment:
Don't think we need the context here. It's just storing the promise so we could store it on the module directly like `contactManipulationPromise`